config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+    "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+    "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<!--
+This file is based on
+https://github.com/google/auto/blob/0c06a2345f71f053714d37bb6549d3460c999f2d/checkstyle.xml
```

Comment:
That's a github page with the contents you want.
Not sure if that is better or worse than the content we want directly, i.e. something like https://raw.githubusercontent.com/google/auto/0c06a2345f71f053714d37bb6549d3460c999f2d/checkstyle.xml